JSDemos/Demos/DateRangeBox/Overview/Angular/app/app.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, NgModule, enableProdMode } from '@angular/core';
+import { BrowserModule, BrowserTransferStateModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { DxDateRangeBoxModule } from 'devextreme-angular';
+
+if (!/localhost/.test(document.location.host)) {
+  enableProdMode();
+}
+
+const msInDay = 1000 * 60 * 60 * 24;
+const now = new Date();
+const initialValue: [Date, Date] = [new Date(now.getTime() - msInDay * 3), new Date(now.getTime() + msInDay * 3)];
+
+function convertRangeToDays(value) {
```

Comment:
Let's make this code more explicit
```suggestion
function convertRangeToDays([startDate, endDate]) {
```